### PR TITLE
Setup script

### PR DESCRIPTION
**Pull request by tunahorse**
Made a small bash script to get going right away. Windows presents some issues, for linux only now.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+
+# Install and upgrade
+echo "Installing build-essential..."
+apt-get update
+apt-get install -y build-essential
```
Comment (danwritecode) [inline start.sh:7]: Could you remove -y? Reason in next comment.

Comment (danwritecode) [inline start.sh:7]: I think we should probably remove all "-y" and explicitly tell the user "You don't have xyz installed, do you want to install it now?"...claude suggests something like this to check if someone has build-essentials installed: 

```bash
if ! dpkg -s build-essential &> /dev/null; then
    echo "build-essential is not installed. Please install it using: sudo apt-get install build-essential"
    exit 1
fi
```

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+if ! command_exists tmux; then
+    echo "tmux is not installed. Installing tmux..."
+    apt-get install -y tmux
```
Comment (danwritecode) [inline start.sh:11]: Same thing here but you can just do `which tmux` to see if it's installed

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+fi
+
+#check if we need to clone
+if [ ! -d "clings" ]; then
+    echo "clings directory not found. Cloning the repository..."
+    git clone https://github.com/danwritecode/clings.git
```
Comment (danwritecode) [inline start.sh:17]: should probably check if they have Git installed also with `which git`

**Review by danwritecode — CHANGES_REQUESTED** [review]
Left a few comments, also, we should probably call this "start_debian.sh"

I use ubuntu so I won't be able to test any other distros, but at least we can start by supporting debian based distros.

**Comment by tunahorse** [comment]
Updated !

**Review by danwritecode — APPROVED** [review]

**Comment by danwritecode** [comment]
this is awesome!!! merging now

**Pull request merged**